src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditorMixins.java

```diff
@@ -34,6 +34,14 @@ public AceEditorMixins(AceEditor editor)
       RStudioGinjector.INSTANCE.injectMembers(this);
       
       editor_ = editor.getWidget().getEditor();
+      timer_ = new Timer()
```

Comment:
This timer was added because I noticed a litany of callbacks being invoked whenever clicking and dragging in the Ace editor surface (since each Ace update would trigger a callback here). Having a timer allows us to avoid the excessive calls.